\documentclass[11pt,]{article}
\usepackage[sc, osf]{mathpazo}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1in]{geometry}




\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

% Now begins the stuff that I added.
% ----------------------------------

% Custom section fonts
\usepackage{sectsty}
\sectionfont{\rmfamily\mdseries\large\bf}
\subsectionfont{\rmfamily\mdseries\normalsize\itshape}


% Make lists without bullets
%\renewenvironment{itemize}{
%  \begin{list}{}{
%    \setlength{\leftmargin}{1.5em}
%  }
%}{
%  \end{list}
%}


% Make parskips rather than indent with lists.
\usepackage{parskip}
\usepackage{titlesec}
\titlespacing\section{0pt}{12pt plus 4pt minus 2pt}{4pt plus 2pt minus 2pt}
\titlespacing\subsection{0pt}{12pt plus 4pt minus 2pt}{4pt plus 2pt minus 2pt}

% Use fontawesome. Note: you'll need TeXLive 2015. Update.
\usepackage{fontawesome}

% Fancyhdr, as I tend to do with these personal documents.
\usepackage{fancyhdr,lastpage}
\pagestyle{fancy}
\renewcommand{\headrulewidth}{0.0pt}
\renewcommand{\footrulewidth}{0.0pt}
\lhead{}
\chead{}
\rhead{}
\lfoot{
\cfoot{\scriptsize  Harrison Specht - CV - \url{https://orcid.org/0000-0003-3151-6803} }}
\rfoot{\scriptsize \thepage/{\hypersetup{linkcolor=black}\pageref{LastPage}}}

% Always load hyperref last.
\usepackage{hyperref}
\PassOptionsToPackage{usenames,dvipsnames}{color} % color is loaded by hyperref

\hypersetup{unicode=true,
            pdftitle={Harrison Specht:  CV (Curriculum Vitae)},
            pdfauthor={Harrison Specht},
            pdfkeywords={RMarkdown, academic CV, template},
            colorlinks=true,
            linkcolor=blue,
            citecolor=Blue,
            urlcolor=blue,
            breaklinks=true, bookmarks=true}
\urlstyle{same}  % don't use monospace font for urls

\begin{document}


\centerline{\huge \bf Harrison Specht}

\vspace{2 mm}

\hrule

\vspace{2 mm}

\moveleft.5\hoffset\centerline{Post-doctoral researcher, Bioengineering,
Northeastern University}
\moveleft.5\hoffset\centerline{360 Huntington Ave, Boston, MA 02110}
\moveleft.5\hoffset\centerline{ \faEnvelopeO \hspace{1 mm} \href{mailto:}{\tt \href{mailto:hms89@cornell.edu}{\nolinkurl{hms89@cornell.edu}}} \hspace{1 mm}     \faGlobe \hspace{1 mm} \href{http://harrisonspecht.com}{\tt harrisonspecht.com}    | \emph{Updated:} \today}

\vspace{2 mm}

\hrule


\hypertarget{education}{%
\section{EDUCATION}\label{education}}

\emph{Northeastern University}, Ph.D.~Bioengineering, Graduate Student
\hfill  March 2022

\emph{Cornell University}, B.A. Chemistry, \emph{magna cum laude}
\hfill 2014

\hypertarget{research-experience}{%
\section{RESEARCH EXPERIENCE}\label{research-experience}}

\emph{Northeastern University}

\begin{quote}
Slavov Lab, Bioengineering \& Barnett Institute, Post-doc
\hfill 2022--Present
\end{quote}

\begin{quote}
Slavov Lab, Bioengineering \& Barnett Institute, Graduate Student
\hfill 2016--2022
\end{quote}

\emph{Merck, Sharpe \& Dohme Corporation}

\begin{quote}
Merck Exploratory Sciences Division, Coop \hfill Spring 2020
\end{quote}

\emph{Broad Institute of MIT and Harvard}

\begin{quote}
Proteomics Platform, Research Associate \hfill 2014--2016
\end{quote}

\emph{Cornell University, Ithaca, NY}

\begin{quote}
Petersen Lab, Department of Chemistry, Research Assistant
\hfill 2012--2014
\end{quote}

\begin{quote}
Zinder Lab, Department of Microbiology, Research Assistant
\hfill 2010--2011
\end{quote}

\begin{quote}
The Triple Helix, magazine, Executive Editor-in-chief \hfill 2013 --
2014
\end{quote}

\emph{Bermuda Institute for Ocean Sciences}

\begin{quote}
National Science Foundation REU Fellow \hfill 2013
\end{quote}

\emph{Vertex Pharmaceuticals, Boston, MA}

\begin{quote}
Drug Metabolism and Pharmacokinetics, Internship \hfill 2013
\end{quote}

\hypertarget{fellowships-and-awards}{%
\section{FELLOWSHIPS AND AWARDS}\label{fellowships-and-awards}}

\textbf{2022} \emph{The AM Prototype Fund} Chosen by the staff of the
D'Amore-McKim School of Business to aid development of single cell
proteomics venture

\textbf{2021} \emph{NSF I-Corps Grant} \$10,000 for development of
single cell proteomics venture

\textbf{2021} \emph{Frequency Bio by Pillar VC \& Petri}
pillar.vc/frequency/

\textbf{2020} \emph{Rising Stars in Proteomics and Metabolomics},
Journal of Proteome Research.

\begin{quote}
\url{https://pubs.acs.org/doi/full/10.1021/acs.jproteome.0c01026}
\end{quote}

\textbf{2019} \emph{2nd place in Chemical Biology}, Royal Society of
Chemistry Twitter Poster Conference

\textbf{2016--} \emph{College of Engineering Dean's Distinguished
Fellowship}, Northeastern University

\textbf{2013} \emph{Best Oral Presentation}, Bermuda Institute for Ocean
Science NSF REU Program

\hypertarget{expertise}{%
\section{EXPERTISE}\label{expertise}}

\textbf{Computational:} Proteomics data processing (DIA, DDA, phospho-,
targeted-), transcriptomics and proteomics data integration, protein-set
enrichment analysis (PSEA), gene-set enrichment analysis (GSEA),
imputation, principal component analysis (PCA), created first
computational single cell proteomics data processing pipeline
(\url{https://github.com/SlavovLab/SCoPE2/}, replicated by `scp'
package:
\url{https://bioconductor.org/packages/release/bioc/html/scp.html})

\textbf{Coding and Software:} R, Shiny, MS-Fragger, DIA-NN, SpectroNaut,
MaxQuant, Skyline

\textbf{Experimental:} Single-cell proteomics, triple-quad and
orbitrap-based mass spectrometry proteomics, data dependent (DDA) and
data independent (DIA) acquisition, protein and peptide N-terminomics,
phosphoproteomics, antibody and chemical enrichment of proteins and
peptides, reverse-phase and ERLIC liquid chromatography, robotic sample
preparation, cell culture, flow cytometry

\hypertarget{publications}{%
\section{PUBLICATIONS}\label{publications}}

Derks, J., Leduc, A., Huffman, R.G., \textbf{Specht, H.}, Ralser, M.,
Demichev, V., Slavov, N. Increasing the throughput of sensitive
proteomics by plexDIA. biorxiv. 2021.
\url{https://doi.org/10.1101/2021.11.03.467007}

Petelski, A.A., Emmott, E., Leduc, A., Huffman, R.G., \textbf{Specht,
H.}, Perlman, D.H., Slavov, N. Multiplexed single-cell proteomics using
SCoPE2. Nature Protocols. doi:
\url{https://doi.org/10.1101/2021.03.12.435034}

\textbf{Specht, H.}, Emmott, E., Petelski, A.A. et al.~Single-cell
proteomic and transcriptomic analysis of macrophage heterogeneity using
SCoPE2. Genome Biol 22, 50 (2021).
\url{https://doi.org/10.1186/s13059-021-02267-5}

\textbf{Specht, H.}, Slavov, N., 2021. Optimizing accuracy and depth of
protein quantification in experiments using isobaric carriers. J.
Proteome Res. 20, 880-887.
\url{https://doi.org/10.1021/acs.jproteome.0c00675}

Keshishian et al.~Highly multiplexed quantitative phosphosite assay for
biology and preclinical studies. bioRxiv 2020.12.08.415281; doi:
\url{https://doi.org/10.1101/2020.12.08.415281} \textbf{In press at
Molecular Systems Biology}

Huffman, R.G., Chen, A., \textbf{Specht, H.}, Slavov, N., 2019. DO-MS:
Data-Driven Optimization of Mass Spectrometry Methods. J. Proteome Res.
18, 2493--2500. \url{https://doi.org/10.1021/acs.jproteome.9b00039}

\textbf{Specht, H.}, Harmange, G., Perlman, D.H., Emmott, E., Niziolek,
Z., Budnik, B., Slavov, N., 2018. Automated sample preparation for
high-throughput single-cell proteomics. bioRxiv 399774.
\url{https://doi.org/10.1101/399774}

\textbf{Specht, H.}, Slavov, N., 2018. Transformative Opportunities for
Single-Cell Proteomics. J. Proteome Res. 17, 2565--2571.
\url{https://doi.org/10.1021/acs.jproteome.8b00257}

Khajuria, R.K., Munschauer, M., Ulirsch, J.C., Fiorini, C., Ludwig,
L.S., McFarland, S.K., Abdulhay, N.J., \textbf{Specht, H.}, Keshishian,
H., Mani, D.R., Jovanovic, M., Ellis, S.R., Fulco, C.P., Engreitz, J.M.,
Sch√ºtz, S., Lian, J., Gripp, K.W., Weinberg, O.K., Pinkus, G.S., Gehrke,
L., Regev, A., Lander, E.S., Gazda, H.T., Lee, W.Y., Panse, V.G., Carr,
S.A., Sankaran, V.G., 2018. Ribosome Levels Selectively Regulate
Translation and Lineage Commitment in Human Hematopoiesis. Cell 173,
90-103.e19. \url{https://doi.org/10.1016/j.cell.2018.02.036}

Keshishian, H., Burgess, M.W., \textbf{Specht, H.}, Wallace, L.,
Clauser, K.R., Gillette, M.A., Carr, S.A., 2017. Quantitative,
multiplexed workflow for deep analysis of human blood plasma and
biomarker discovery by mass spectrometry. Nat Protoc 12, 1683--1701.
\url{https://doi.org/10.1038/nprot.2017.054}

\hypertarget{talks}{%
\section{TALKS}\label{talks}}

\textbf{2022}. \textbf{Harrison Specht}. The Scientist, webinar series:
\href{https://www.the-scientist.com/sponsored-webinars/using-single-cell-proteomics-to-understand-human-health-and-disease-69977}{Accessible
single cell proteomics by mass spectrometry}. \emph{\textbf{Invited}}

\textbf{2022}. \textbf{Harrison Specht}. Accessible single cell
proteomics by mass spectrometry. The Broad Institute of MIT and
Harvard's Cell Circuits and Epigenomics seminar series.
\emph{\textbf{Invited}}

\textbf{2020}. \textbf{Harrison Specht}, Edward Emmott, Aleksandra A.
Petelski, R. Gray Huffman, David H. Perlman, Marco Serra, Peter
Kharchenko, Antonius Koller, Nikolai Slavov. ``How to perform
quantitative single cell proteomics with SCoPE2.'' GenomeWeb Webinar
Series. \emph{\textbf{Invited}}

\textbf{2020}. \textbf{Harrison Specht}, Edward Emmott, Aleksandra A.
Petelski, R. Gray Huffman, David H. Perlman, Marco Serra, Peter
Kharchenko, Antonius Koller, Nikolai Slavov. ``How to perform
quantitative single cell proteomics with SCoPE2.'' Association of
Biomolecular Resource Facilities (ABRF) 2020 Annual Meeting.
\emph{\textbf{Invited}}

\textbf{2019}. \textbf{Harrison Specht}, Edward Emmott, Aleksandra A.
Petelski, R. Gray Huffman, David H. Perlman, Antonius Koller, Nikolai
Slavov. ``Design of single cell proteomics experiments.'' Single Cell
Proteomics Conference. Boston, MA. 2019.

\textbf{2019}. \textbf{Harrison Specht}, Nikolai Slavov. Quantifying
proteins by mass spectrometry. Models, Inference, and Algorithms
Seminar. Broad Institutde of MIT and Harvard. \emph{\textbf{Chalk-talk}}

\textbf{2018}. \textbf{Harrison Specht}, Guillaume Harmange, David H
Perlman, Edward Emmott, Zachary Niziolek, Bogdan Budnik, Nikolai Slavov.
``Automated sample preparation for high-throughput single-cell
proteomics.'' HUPO 2018.

\textbf{2018}. \textbf{Harrison Specht}, Guillaume Harmange, David H
Perlman, Edward Emmott, Zachary Niziolek, Bogdan Budnik, Nikolai Slavov.
``Automated sample preparation for high-throughput single-cell
proteomics.'' Single Cell Proteomics Conference. Boston, MA. 2018.

\hypertarget{posters}{%
\section{POSTERS}\label{posters}}

\textbf{2019}. \textbf{Harrison Specht}. ``Automated sample preparation
for high-throughput single-cell proteomics.'' Royal Society of Chemistry
Twitter Poster Conference. \emph{\textbf{Twitter poster presenter}}
Awarded 2nd place in Chemical Biology category.

\textbf{2018}. Hasmik Keshishian, Luke Wallace, \textbf{Harrison
Specht}, Judit Jan-Valbuena, Rob McDonald, Dale Petterson, Eric Kuhn,
Michael Burgess, D. R. Mani, Tomas Rejtar, Javad Golji, Karen Wang,
William Sellers, Steven A. Carr. ``SigPath300: A high throughput
MS-based assay to quantify over 300 phosphosites of known biological
relevance in cells and tissues.'' American Society for Mass Spectrometry
Annual Meeting. \emph{\textbf{Contributor}}

\textbf{2017}. Bogdan Budnik, Ezra Levy, \textbf{Harrison Specht},
Nikolai Slavov. ``Exploring cell division dynamics across single cell
proteomes.'' Cold Spring Harbor Laboratories: Single Cell Analyses.
\emph{\textbf{Poster Presenter}}

\textbf{2017}. \textbf{Harrison Specht}, Ezra Levy, Bogdan Budnik,
Nikolai Slavov. ``Mass-spectrometry of single mammalian cells quantifies
proteome heterogeneity during cell differentiation.'' American Society
for Mass Spectrometry Annual Meeting. \emph{\textbf{Poster Presenter}}

\textbf{2015}. \textbf{Harrison Specht}, Hasmik Keshishian, Rajiv K.
Khajuria, et al.~``Addressing the challenge of ribosomal protein
stoichiometry by proteomics.'' Broad Institute of MIT and Harvard
Retreat. \emph{\textbf{Poster Presenter}}

\textbf{2013}. \textbf{Harrison Specht} and Andrew Peters
``Investigating North Atlantic Tar in Bermuda: the hydrocarbon
composition of North Atlantic pelagic tar and their hydrocarbon leaching
into seawater.'' BIOS NEF REU Program. \emph{\textbf{Presenter}}

\hypertarget{professional-affiliations-training-service}{%
\section{PROFESSIONAL AFFILIATIONS, TRAINING \&
SERVICE}\label{professional-affiliations-training-service}}

\begin{itemize}
\tightlist
\item
  Volunteer for Single Cell Proteomics Conference 2018, 2019, 2021
\item
  Volunteer moderator for Single Cell Proteomics Conference (Zoom) 2020
\item
  Northeastern University Graduate Student Liason, 2018-2019
\item
  Member of Northeastern University Fencing Club
\item
  Member of Barnett Institute for Chemical and Biological Analysis
\item
  eLife Ambassador, 2018
\item
  ASAPbio Ambassador, 2018
\item
  American Society for Mass Spectrometry
\item
  May Institute on computation and statistics for mass spectrometry and
  proteomics, 2016
\item
  Lab preprint journal club founder
\end{itemize}

\hypertarget{references}{%
\section{REFERENCES}\label{references}}

Available upon request

\end{document}
